clans/src/main/java/me/mykindos/betterpvp/clans/tips/tips/ClanSetHomeTip.java

```diff
@@ -0,0 +1,37 @@
+package me.mykindos.betterpvp.clans.tips.tips;
```

Comment:
Originally ClanHomeTip, moved to new file as ClanHomeTip now refers to /c home instead of /c sethome